### PR TITLE
Routing refactoring

### PR DESCRIPTION
**Pull request by YaroslavNikiforov**

### CODE REVIEW

**Comment by YaroslavNikiforov** [comment]
[Remove routing from index.js](https://trello.com/c/Wajbdjvf/252-remove-routing-from-indexjs)

**Comment by YaroslavNikiforov** [comment]
[Refactoring authService (the begining)](https://trello.com/c/24NSphFm/234-refactoring-authservice-the-begining)

**Comment by YaroslavNikiforov** [comment]
[Refactoring LoginPage](https://trello.com/c/9bn3JbYx/250-refactoring-loginpage)

File: front-end/src/components/Admin/Record/RecordEditor/index.js [hunk 1]
```diff
@@ -63,6 +64,10 @@ class RecordEditor extends Component {
           loading: false
         });
         toastr.success('Данные успешно сохранены');
+
+        for (let i = 0; i < formInputs.length; i++) {
```
Comment (Thealoner) [inline front-end/src/components/Admin/Record/RecordEditor/index.js]: Consider moving both resizeInputs from two functions to a single place in componentDidUpdate.

File: front-end/src/components/Admin/Record/RecordEditor/index.js [hunk 2]
```diff
@@ -99,6 +104,10 @@ class RecordEditor extends Component {
         this.props.history.push({
           pathname: '/admin/' + tabKey + '/' + editorUrl
         });
+
+        for (let i = 0; i < formInputs.length; i++) {
+          resizeInput(formInputs[i]);
+        }
```
Comment (Thealoner) [inline front-end/src/components/Admin/Record/RecordEditor/index.js]: It would be nice, if you could change the resizeInput function, so that you can pass formInputs array to it:
resizeInput(formInputs);
and it would resize all inputs from it. So that you do wouldn't need to write the cycle every time.
You can do this in a separate PR, if you wish.

File: front-end/src/components/Header/User/index.js [hunk 3]
```diff
@@ -1,20 +1,18 @@
 import React, {Component} from 'react';
 import './index.scss';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import photo from './user-avatar.png';
```
Comment (Thealoner) [inline front-end/src/components/Header/User/index.js]: I'm adding these spaces and you're removing them :) This could go on forever ))
